random_menu.rb

```diff
@@ -0,0 +1,36 @@
+
+
+# Create three arrays. One wiht adjectives, one with cooking-style and one with food.
+adjectives = ["hot", "blend", "cold", "warm", "spicy",
+  "sweet", "sweet & sour", "bitter", "creamy", "crunchy"]
+cooking_style = ["cooked", "steamed", "grilled", "boiled", "seared",
+  "baked", "pan-fried", "oil-fried", "simmering", "marinated"]
+food = ["beef", "vegetables", "cookies", "lamb", "pasta",
+  "chicken", "salmon", "shrimps", "clams", "dumplings"]
+
+descriptive_terms = [adjectives, cooking_style, food]
+
+# Input of how many items the menue will have
+print "How many items would you like to see on the menue? "
+number_of_items = gets.chomp.to_i
+item_number = 0
+
+# Printing out the items one by one
+number_of_items.times do
```

Comment:
Note that a `times` loop gives you the current count as an iteration variable. So this loop could be rewritten as:

```ruby
number_of_items.times do |n|
  # ...get adj_choise, style_choise and food_choise...
  puts "#{n+1}, #{adj_choise} #{style_choise} #{food_choise}"
end
```

Both ways will work, so do what makes sense for you.